app/forms/assessor_interface/work_history_contact_form.rb

```diff
@@ -11,6 +11,8 @@ class AssessorInterface::WorkHistoryContactForm
 
   validates :work_history, :user, presence: true
 
+  validates :email, valid_for_notify: true
```

Comment:
I've had a good search for valid_for_notify but couldn't find any documentation, what does this do?

Reply:
ok, I think figured it out, it's a custom validation ValidForNotifyValidator. But when it's called you leave of Validator as convention?